sdv/datasets/local.py

```diff
@@ -42,3 +42,48 @@ def load_csvs(folder_name, read_csv_parameters=None):
         )
 
     return csvs
+
+
+def save_csvs(data, folder_name, suffix=None, to_csv_parameters=None):
+    """Save dataframes to csv files in a specified folder.
+
+    Args:
+        data (dict):
+            A dictionary that maps each table name (string) to the data for that
+            table (pandas.DataFrame).
+        folder_name (str):
+            The full path of the folder where the data will be saved.
+        suffix (str):
+            A string to be appended to the name of the file. Defaults to ``None``.
+        to_csv_parameters (dict):
+            A python dictionary of with string and value accepted by ``pandas.DataFrame.to_csv``
+            function. Defaults to ``None``.
+    """
+    if not path.exists(folder_name):
+        raise ValueError(f"The folder '{folder_name}' cannot be found.")
```

Comment:
```
(required) folder_name: The name of the folder to write the CSVs into. If the folder does not exist, we will create the folder
```
Instead of raising the error we have to create the folder.

Reply:
Good catch, thanks, done in [fab0aae](https://github.com/sdv-dev/SDV/pull/1859/commits/fab0aaed1dc1d68954cb43d89512b95e297779b7)